doc/conf.py

```diff
@@ -24,14 +21,21 @@
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
 
+# The following ensures that astropy-helpers is downloaded from PyPI and
+# installed locally in this directory, since it is needed for the Sphinx
+# extensions.
+from setuptools import Distribution
+Distribution({'setup_requires': 'astropy-helpers'})
```

Comment:
In the end we do install astropy-helpers for travis, this is just for users who want to build locally. I thought we didn't want people to have astropy-helpers installed as a package? (and most users aren't familiar with virtualenv)